### ISSUE TITLE
Re-add stock AppSerializer. Experiment with `App::build()`.

### ISSUE THREAD

**Issue by beporter** [loadsys/CakePHP-Serializers#109]
[The book](http://book.cakephp.org/2.0/en/core-utility-libraries/app.html#add-new-packages-to-an-application) makes it sound like you should be able to control the order in which paths are checked. We want `Serializer/` checked first, and `Plugin/Serializers/Serializer/` to be checked last. When this order is respected, we can add a stock `AppSerializer` back into the package and not force projects to create their own in order for the plugin to work.

**Comment by beporter** [comment]
Did we figure this out yet? I remember we completed a "second pass" on this already, but I can't recall if a third pass was still necessary.

**Comment by jtyost2** [comment]
No, we haven't currently. AppSerializers is only documented, without a sample anywhere in the project.
